gpii/node_modules/flowManager/configs/gpii.flowManager.config.production.json

```diff
@@ -18,7 +18,7 @@
                 "priority": "after:preferencesDataSource.productionDataSource"
             },
             "distributePort.productionPort": {
-                "record": 8081,
+                "record": 8082,
```

Comment:
Does this change in port number have consequences for usages (CI, production deployments) of this configuration?

Reply:
It looks like the CI build failed:

    01:21:19.708:  jq: FAIL: Module "gpii.config.trusted-flowmanager.production tests" Test name "Production config test with os_common using Flat matchmaker" - Message: Error making request to /user/MikelVargas/login: connect ECONNREFUSED 127.0.0.1:8081

https://ci.gpii.net/job/universal-node-production-tests/488/console